foreman-selinux.spec

```diff
@@ -0,0 +1,129 @@
+# vim: sw=4:ts=4:et
+#
+# Copyright (c) 2013 Red Hat, Inc.
+
+# This program and entire repository is free software: you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation, either version 3 of the License,
+# or any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+%define selinux_variants targeted
+%define selinux_modules katello
+
+%if 0%{?rhel} == 5
+# absolute minimum versions for RHEL 5
+%define selinux_policy_ver 3.11.1-81
+%else
+# absolute minimum versions for RHEL 6
+%define selinux_policy_ver 2.4.6-80
+%endif
+%define selinux_policycoreutils_ver 1.33.12-1
+
+%define moduletype apps
+
+# set and uncomment all three to set alpha tag
+#global alphatag RC1
+#global dotalphatag .%{alphatag}
+#global dashalphatag -%{alphatag}
+
+Name:           katello-selinux
+Version:        2.1.0
+Release:        0%{?dotalphatag}%{?dist}
+Summary:        SELinux policy module for katello
+
+Group:          System Environment/Base
+License:        GPLv3+
+URL:            http://www.katello.org
+Source0:        http://downloads.theforeman.org/%{name}/%{name}-%{version}%{?dashalphatag}.tar.bz2
+
+BuildRequires:  checkpolicy, selinux-policy-devel, hardlink
+BuildRequires:  policycoreutils >= %{selinux_policycoreutils_ver}
+BuildRequires:  /usr/bin/pod2man
+BuildArch:      noarch
+
+Requires:           selinux-policy >= %{selinux_policy_ver}
+Requires(post):     /usr/sbin/semodule, /sbin/restorecon, /usr/sbin/setsebool, /usr/sbin/selinuxenabled, /usr/sbin/semanage
+Requires(post):     policycoreutils-python
+Requires(post):     selinux-policy-targeted
+Requires(postun):   /usr/sbin/semodule, /sbin/restorecon
```

Comment:
We need it for elasticsearch port definition which will prevent *foreman* policy from loading (katello is not the problematic one in this case). See the referenced PR. We will likely bump the release number to 1.8.0-1 to get there.